http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -352,10 +354,21 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
         .map(_.getOrElse(List.empty))
 
       // We must also check the RuntimeSamResourceId in sam to support already existing and newly created google runtimes
-      samVisibleRuntimeSamResourceIds <- NonEmptyList
-        .fromList(runtimesUserIsNotCreator.map(_.samResource))
-        .traverse(ids => authProvider.filterUserVisible(ids, userInfo))
-        .map(_.getOrElse(List.empty))
+      runtimesAndProjects = runtimesUserIsNotCreator.map(r => (GoogleProject(r.cloudContext.asString), r.samResource))
```

Comment:
`(GoogleProject(r.cloudContext.asString)` doesn't seem right cuz you the cloud context could be either GCP or Azure

this doesn't seem will work for runtimes created for v2 workspace (and it won't work for Azure)

Reply:
this is the fallback for runtimes created with the v1 endpoint... the `GoogleProject` cast is for compatibility with the `authProvider`